src/apis/subscriber.ts

```diff
@@ -3,8 +3,50 @@ import api from '@/utils/api';
 export const getSubscribers = async () => {
   const response = await api.get('/api/subscriber');
   const data = await response.data;
-  {
-    console.log(data, '-->checking subscriber data');
-  }
   return data;
 };
+
+export const createSubscriber = async (subscriber: any) => {
+  try {
+    const response = await api.post('/api/subscriber', subscriber);
+    const data = await response.data;
+    return data;
+  } catch (error: unknown) {
+    throw error;
+  }
+};
+
+export const updateSubscriber = async (
+  subscriberId: number,
+  subscriber: any
```

Comment:
is the following possible here?
```suggestion
  subscriber: Subscriber
```